homeassistant/const.py

```diff
@@ -148,6 +148,7 @@
 CONF_STATE = 'state'
 CONF_STRUCTURE = 'structure'
 CONF_SWITCHES = 'switches'
+CONF_SLOTS = 'slots'
```

Comment:
new config to define slot's friendly names

Reply:
Let's keep this in the local file instead of adding it to the general constants.